web/web.go

```diff
@@ -83,23 +83,30 @@ func CheckBrowserCompatability(c *api.Context, r *http.Request) bool {
 	return true
 }
 
-func root(c *api.Context, w http.ResponseWriter, r *http.Request) {
-	if !CheckBrowserCompatability(c, r) {
-		w.Header().Set("Cache-Control", "no-store")
-		page := utils.NewHTMLTemplate(c.App.HTMLTemplates(), "unsupported_browser")
-		page.Props["Title"] = c.T("web.error.unsupported_browser.title")
-		page.Props["Message"] = c.T("web.error.unsupported_browser.message")
-		page.RenderToWriter(w)
-		return
+func root() func(c *api.Context, w http.ResponseWriter, r *http.Request) {
+	staticDir, _ := utils.FindDir(model.CLIENT_DIR)
```

Comment:
I like that we're only going to be looking up the client directory once. That seems like it might be a bit of a performance improvement